services/app-api/email/CMSSubsequentSubmissionNotice.js

```diff
@@ -7,6 +7,7 @@ import { getCPOCandSRTEmailAddresses } from "../utils/getCpocAndSrtEmail.js";
  * @returns {Object} email parameters in generic format.
  */
 export const CMSSubsequentSubmissionNotice = async (data, config) => {
+  data.submitterName = ""; // remove this bc we dont want it on the cms email
```

Comment:
do we want to remove this line?

Reply:
It does not look like it is on the template so I would say yes. To absolutely sure, you can check with Hanna if that should be there.